dashboard/app/reporting.go

```diff
@@ -900,6 +900,49 @@ func queryCrashesForBug(c context.Context, bugKey *db.Key, limit int) (
 	return crashes, keys, nil
 }
 
+func queryJobsForBug(c context.Context, bugKey *db.Key, jobType JobType) (
+	[]*Job, []*db.Key, error) {
+	var jobs []*Job
+	keys, err := db.NewQuery("Job").
+		Ancestor(bugKey).
+		Filter("Type=", jobType).
+		Filter("Finished>", time.Time{}).
+		Order("-Finished").
+		GetAll(c, &jobs)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to fetch fix bisections: %v", err)
+	}
+	return jobs, keys, nil
+}
+
+func queryCrashForJob(c context.Context, job *Job, bugKey *db.Key) (*Crash, error) {
+	// If there was no crash corresponding to the Job, return.
+	if job.CrashTitle == "" {
+		return nil, nil
+	}
+	crashes, _, err := queryCrashesForBug(c, bugKey, 1)
```

Comment:
queryCrashesForBug returns a wrong crash.
We need the job.CrashID crash.

Reply:
Thanks! I'll update and upload the PR.
In this case we don't have the crash details stored in table, so I'm fetching the latest crash and updating some fields using the crash details inside job. What am I missing?

Reply:
For ReproOpts/ReproSyz/ReproC  we need to take them from job.CrashID. That's exactly the repro that was used during bisection. latestCrash may be a wrong one.

For ReportLen (priority) we should leave it as 0, I don't think it's used anywhere in UI. (or we should could compute a correct value, see saveCrash function, we have all info for this, but it does not seem to be needed).

For Maintainers, I think we can also leave it as empty, it's only used to show in the additional column and I don't think lots of people are interested in this. We don't have the right info for this crash, so showing nothing is better than showing wrong info.

Reply:
I see, thanks, I'll update the PR.